image-sync-formula/image-sync-formula.changes

```diff
@@ -1,3 +1,13 @@
+-------------------------------------------------------------------
+Thu May  4 12:47:11 UTC 2023 - Vladimir Nadvornik <nadvornik@suse.cz>
+
+- Delete cache files that are no longer needed
+
+-------------------------------------------------------------------
+Thu May  4 11:01:54 UTC 2023 - Vladimir Nadvornik <nadvornik@suse.cz>
+
```

Comment:
Can you add this as one entry? Last time I tried to submit like this it was rejected by autobuild team.